testing/test_aux.out

```diff
@@ -1,10 +1,12 @@
-Running testing/test_aux.sh
+Running ./testing/test_aux.sh
```

Comment:
This shouldn't change, which means you're likely running it wrong.  How are you running it?  You should just do:

"sudo testing/test_aux.sh" -- my guess is you added in an extra "./" to the path.

Reply:
Still needs to be fixed.